src/feature-test/user_see_ingredient_after_add.test.js

```diff
@@ -0,0 +1,40 @@
+import '../setupTest.js'
+
+describe('I.I.A.S', () => {
+  it('User wants to see what they are about to submit', async () => {
+    const browser = await puppeteer.launch({
+      // This two lines below allows the dev to see the flow on the chromium page
```

Comment:
Mentioned this on @bengscott2 also.
I've removed all comments.

Reply:
ok I'll take them all out